includes/templates/admin/edit-story.php

```diff
@@ -35,8 +35,9 @@
 
 // Preload common data.
 $preload_paths = [
-	sprintf( '/web-stories/v1/%s/%s?context=edit', $rest_base, $post->ID ),
+	sprintf( '/web-stories/v1/%s/%s?context=edit&_embed=author&_locale=user', $rest_base, $post->ID ),
 	'/web-stories/v1/media?context=edit&per_page=100&page=1&_web_stories_envelope=true',
+	'/web-stories/v1/users?per_page=100&who=authors&_locale=user',
```

Comment:
@swissspidy Could you double-check this since it's a PHP file?

Reply:
First things, first, I do not think we should preload this data. Preloaded data is only for critical that stop editor from rendering. As the user dropdown is not on screen, that is the not case. We would stop the render of the story to load this data, that some users ( author level ) can't assign another user to a user, making it pointless to load. 

Second, as noted above, this would only get the first page of results for users. This is not what we want, we want all the users.

Reply:
```suggestion
	'/web-stories/v1/users?per_page=100&who=authors',
```

In testing, I saw REST API requests. This fixed it.

Reply:
@swissspidy 👆🏻

Reply:
Removed the `_locale` bit now.

The preloading like this makes sense to me though, as it matches the editor behavior. We don't to load all users, just the initial set of data for the dropdown, which is 100 users.